app/src/main/java/com/hedvig/app/feature/insurance/ui/InsuranceFragment.kt

```diff
@@ -57,8 +78,8 @@ class DashboardFragment : Fragment(R.layout.fragment_dashboard) {
             }
         }
 
-        (root.adapter as? DashboardAdapter)?.items =
-            listOf(DashboardModel.Header) + contracts + upsells
+        (root.adapter as? InsuranceAdapter)?.items =
+            listOf(InsuranceModel.Header) + infoBoxes + contracts + upsells
```

Comment:
Does this still hold? Shouldn't the renewal info boxes be associated with each contract?

Reply:
We are getting them from the contract with `val renewals = dashboardData?.contracts.orEmpty().mapNotNull { it.upcomingRenewal }`

Reply:
Yes, but I believe they should be interspersed in the designs, not all stacked up at the top 😉